enterprise/signals.py

```diff
@@ -331,7 +331,9 @@ def update_enterprise_catalog_data(sender, instance, **kwargs):     # pylint: di
         else:
             response = catalog_client.get_enterprise_catalog(catalog_uuid=catalog_uuid)
     except NotConnectedToOpenEdX as exc:
-        logger.exception('Unable to update Enterprise Catalog {}'.format(str(catalog_uuid)), exc)
+        logger.exception(
+            'Unable to update Enterprise Catalog {}'.format(str(catalog_uuid)), exc_info=exc
```

Comment:
Nit: I *think* adding `exc_info=exc` here is redundant, because we're calling `logger.exception()`, but it's also harmless, so no big deal either way.

Reply:
ah I guess I am still getting used to this logging setup in python.. but yeah thanks for the note. I will have to delve into my logging training a bit then.